src/libraries/System.Reflection.Emit.ILGeneration/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -1,7 +1,5 @@
 Compat issues with assembly System.Reflection.Emit.ILGeneration:
 MembersMustExist : Member 'public void System.Reflection.Emit.ILGenerator.MarkSequencePoint(System.Diagnostics.SymbolStore.ISymbolDocumentWriter, System.Int32, System.Int32, System.Int32, System.Int32)' does not exist in the reference but it does exist in the implementation.
```

Comment:
I assume MarkSequencePoint, GetMethodSigHelper are still necessary and must be `public` for debugging cases and\or for internal implementation. Is that correct?

Reply:
`MarkSequencePoint` could be removed. `GetMethodSigHelper` overloads were used but could be made internal.